relay-server/src/services/outcome.rs

```diff
@@ -251,6 +252,79 @@ impl fmt::Display for Outcome {
     }
 }
 
+/// A lower-cardinality version of [`RuleId`].
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub enum RuleCategory {
+    BoostLowVolumeProjects,
+    BoostEnvironments,
+    IgnoreHealthChecks,
+    BoostKeyTransactions,
+    Recalibration,
+    BoostReplayId,
+    BoostLowVolumeTransactions,
+    BoostLatestReleases,
+    Custom,
+    Other,
+}
+
+impl RuleCategory {
+    fn to_str(self) -> &'static str {
```

Comment:
```suggestion
    fn as_str(self) -> &'static str {
```

We have `as_str` throughout Relay for enums